test/system/src/test/java/io/pravega/test/system/MultiControllerTest.java

```diff
@@ -140,6 +141,7 @@ public void multiControllerTest() throws ExecutionException, InterruptedExceptio
                     throwable -> throwable instanceof RetriesExhaustedException);
         }
 
+        Futures.getAndHandleExceptions(controllerService.scaleService(1), ExecutionException::new);
```

Comment:
If the change in `isRunning()` is correct, we can remove this line.

Reply:
Reverted that